### DISCUSSION TITLE
Separate instance of Kibana

### DISCUSSION THREAD

**Discussion by pauhaith360** [telekom-security/tpotce#1218] [Q&A]
I am wondering if its possible to set up a separate instance of Kibana to connect to a single tpot elasticsearch instance. So I could have multiple kibana dashboards. I have attempted to clone the repo use the docker files and just docker compose up a kibana instance but it only loads to a blank page. Is there a specific configuration I would need to adjust to have more than one Kibana?

I also did find the info on sending logstash data to another source which worked great. But I was hoping to not have to deal with the overhead of managing multiple elasticsearch  set ups each with big indexes.

**Comment by lcia-projects** [comment]
if i understand your question correctly.. all you'd have to do is open up the elasticsearch ports in the docker .yml file so another kibana instance could access it. Then create your kibana server or instance.. and point to the host address of where your tpot is running.

**Reply by pauhaith360** [reply]
Sorry for my late response. Holiday's and all. I've tried this. I get a error saying that it cannot tell what version of Elasticsearch is running. I think it has something to do with the initial set up of the Kibana and Elastic where there is a key that is passed to Kibana that Elastic generates when it comes up. But I could be wrong. I'll go back through my set up and make sure to try your  suggestion. But overall it looks like I'm just going to have to go with sending logstash data to a separate set up of Kibana and Elastic. Thanks again for the response.
